eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/controller/MetaController.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.console.controller;
+
+import org.apache.eventmesh.dashboard.console.entity.meta.MetaEntity;
+import org.apache.eventmesh.dashboard.console.modle.dto.meta.NewMetaDTO;
+import org.apache.eventmesh.dashboard.console.service.registry.RegistryDataService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MetaController {
+
+    @Autowired
+    private RegistryDataService metaService;
+
+    @PostMapping("/meta/newMeta")
+    public String addRegistry(@Validated @RequestBody NewMetaDTO newMetaDTO) {
+        MetaEntity query = new MetaEntity();
+        query.setName("");
+        query.setType("");
+        query.setVersion("");
+        query.setClusterId(0L);
+        query.setHost(newMetaDTO.getMetaHost());
+        query.setPort(newMetaDTO.getMetaPort());
+        query.setRole("");
+        query.setUsername("");
+        query.setParams("");
+        query.setClusterId(0L);
+
+        metaService.insert(query);
+
+        return "success";
+    }
```

Comment:
I don't think we need to add a Registry on front-end manually.

When adding an EventMesh Cluster, the Registry address and namespace are specified and should be stored into `ClusterEntity` as fields.

When discovering EventMesh Clusters from a Registry address, the Registry address and namespaces which each contains an EventMesh Cluster should be stored into multiple `ClusterEntity`s as fields either.

Because the sum of the registry does not represent the sum of the cluster, it is very difficult to try to store the registry entities and then manage the cluster.

Reply:
you mean, discover the cluster by adding a cluster into database but not adding a meta(now my solution)? Can it be guaranteed that a registration center only manages one cluster?

Reply:
>discover the cluster by adding a cluster into database but not adding a meta

Yes

>Can it be guaranteed that a registration center only manages one cluster?

Of course. To be exact, users have to isolate different clusters with different namespaces (c.f. https://nacos.io/en-us/docs/concepts.html). It is a common best practice.

Reply:
ok

Reply:
fixed in [1f05117](https://github.com/apache/eventmesh-dashboard/pull/98/commits/1f051174a398f0080f30b44680163d1297914c39)